.github/workflows/integration_test.yml

```diff
@@ -0,0 +1,39 @@
+# TODO: Support running all integration tests when it's possible to 
+# run self-hosted runners in parallel.
+name: Integration Test
+
+on:
+  pull_request:
+
+jobs:
+  changed_packages:
```

Comment:
is this correct job name?

Reply:
I named it so the information is sufficient in the UI, but I'm open to other suggestions.
![Screenshot from 2021-09-28 15-47-56](https://user-images.githubusercontent.com/43136596/135037451-5a08cbd8-2fbf-4f5f-8912-e60f398ba032.png)

Reply:
No problem. I was just curious. So, this means "test the changed packages"?

Reply:
Yes, that's correct.